DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/sortingQueryResults/metadata.json

```diff
@@ -0,0 +1,223 @@
+                "",
+                "* See related links for details about ordering by:<br>",
+                "  _Score_, _Aggregated values_, _Spatial distance_, or _Random ordering_."
+            ],
+            "Assets": [
+                {
+                    "Title": "Querying: Sorting",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/querying/sorting"
+                },
+                {
+                    "Title": "Sorting query results",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch09/Ch09.md#sorting-query-results"
+                },
+                {
+                    "Title": "Indexes: Sorting & Collation",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/sorting-and-collation"
+                },
+                {
+                    "Title": "Random Ordering",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/querying/how-to-customize-query#randomordering"
+                },
+                {
+                    "Title": "Order by Distance",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/querying/how-to-query-a-spatial-index#orderbydistance"
+                }
+            ]
+        },
+        {
+            "Title": "Secondary sort by field value",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* _Multiple sorting actions_ can be chained.",
+                "  You can sort by one field and then perform a secondary sort on another field.",
+                "  Any combination of secondary sorting is possible as the index fields are indexed independently of one another.",
+                "",
+                "* There is no limit on the number of sorting actions that can be chained.<br>",
+                "  In the example, a secondary sort is done on the _Name_ field.",
+                "",
+                "* Options:<br>",
+                "  `orderBy()` - Order by ascending values.<br>",
+                "  `orderByDescending()` - Order values in descending order.",
+                "",
+                "* The order is computed according to the inferred field type, with strings defaulting to lexicographic order.",
+                "  In the above example, passing _OrderingType.ALPHA_NUMERIC_ will cause the digit characters in the string to be treated as numbers and not as text.<br>",
+                "",
+                "* You can override the ordering type with any of the following:<br>",
+                "    * OrderingType.String (lexicographic ordering)<br>",
```

Comment:
I suspect the props naming convention is different in java. 
ALPHA_NUMERIC, etc.

Reply:
done